great_expectations/render/types/__init__.py

```diff
@@ -1,497 +1,246 @@
-        return d
+import warnings
+
+from great_expectations.render import CollapseContent as CollapseContentRender
+from great_expectations.render import (
+    RenderedAtomicContent as RenderedAtomicContentRender,
+)
+from great_expectations.render import (
+    RenderedAtomicContentSchema as RenderedAtomicContentSchemaRender,
+)
+from great_expectations.render import RenderedAtomicValue as RenderedAtomicValueRender
+from great_expectations.render import (
+    RenderedAtomicValueGraph as RenderedAtomicValueGraphRender,
+)
+from great_expectations.render import (
+    RenderedAtomicValueSchema as RenderedAtomicValueSchemaRender,
+)
+from great_expectations.render import (
+    RenderedBootstrapTableContent as RenderedBootstrapTableContentRender,
+)
+from great_expectations.render import (
+    RenderedBulletListContent as RenderedBulletListContentRender,
+)
+from great_expectations.render import (
+    RenderedComponentContent as RenderedComponentContentRender,
+)
+from great_expectations.render import RenderedContent as RenderedContentRender
+from great_expectations.render import (
+    RenderedContentBlockContainer as RenderedContentBlockContainerRender,
+)
+from great_expectations.render import (
+    RenderedDocumentContent as RenderedDocumentContentRender,
+)
+from great_expectations.render import RenderedGraphContent as RenderedGraphContentRender
+from great_expectations.render import (
+    RenderedHeaderContent as RenderedHeaderContentRender,
+)
+from great_expectations.render import (
+    RenderedMarkdownContent as RenderedMarkdownContentRender,
+)
+from great_expectations.render import (
+    RenderedSectionContent as RenderedSectionContentRender,
+)
+from great_expectations.render import (
+    RenderedStringTemplateContent as RenderedStringTemplateContentRender,
+)
+from great_expectations.render import RenderedTableContent as RenderedTableContentRender
+from great_expectations.render import RenderedTabsContent as RenderedTabsContentRender
+from great_expectations.render import TextContent as TextContentRender
+from great_expectations.render import ValueListContent as ValueListContentRender
+
+
+# TODO: Remove this entire module for release 0.18.0
+def _get_deprecation_warning_message(classname: str) -> str:
+    return (
+        f"Importing the class {classname} from great_expectations.render.types is deprecated as of v0.15.32 "
+        f"in v0.18. Please import class {classname} from great_expectations.render."
+    )
+
+
+class CollapseContent(CollapseContentRender):
+    # deprecated-v0.15.32
+    classname = "CollapseContent"
+    warnings.warn(
+        _get_deprecation_warning_message(classname=classname),
+        DeprecationWarning,
+    )
+
+
+class RenderedAtomicContent(RenderedAtomicContentRender):
+    # deprecated-v0.15.32
+    classname = "RenderedAtomicContent"
+    warnings.warn(
+        _get_deprecation_warning_message(classname=classname),
+        DeprecationWarning,
+    )
```

Comment:
So will all these warnings run when this `types` package is imported?

Reply:
No they would not have. Updated to add a test and leverage `__init__`.